ABCall.swagger.yaml

```
@@ -97,6 +97,92 @@ paths:
         address: "${client_url}"
         protocol: h2
         path_translation: APPEND_PATH_TO_ADDRESS
+  /v1/employees:
+    post:
+      summary: Create employee
+      deprecated: false
+      description: >-
+        Creates a new employee without associating it with a specific client
+        initially. Employees can be administrators, analysts, or agents. 
```

yamllint

[error] trailing spaces

(trailing-spaces)